db/controller.js

```diff
@@ -10,4 +10,14 @@ const getCategories = (request, response, next) => {
       });
 };
 
-module.exports = { getCategories };
\ No newline at end of file
+const getReviews = (request, response, next) => {
+
+  fetchReviews().then((reviews) => {
+    response.status(200).send({ reviews });
+  })
+  .catch((err) => {
+    console.log(err)
```

Comment:
you are passing in 'next' to the function but not using it - something to think about when error handling :)

Reply:
cool, will add this in for my next pull request